crates/ruma-events/src/dummy.rs

```diff
@@ -19,7 +19,8 @@ use serde::{
 /// this *m.dummy* event as the most recent event and using the keyshare request to set up the
 /// session. The keyshare request and *m.dummy* combination should result in the original sending
 /// client receiving keys over the newly established session.
-#[derive(Clone, Debug, EventContent)]
+#[derive(Clone, Debug, Default, EventContent)]
+#[allow(clippy::exhaustive_structs)]
 #[ruma_event(type = "m.dummy", kind = ToDevice)]
 pub struct DummyToDeviceEventContent;
```

Comment:
Can you actually re-add the constructor? Just for consistency with other event content types.